docs/contentSharing.md

```diff
@@ -81,9 +81,18 @@ Just as anyone with access to a question in your course can access any file in `
 
 ## Sharing assessments
 
-You may also publicly share whole assessments, provided that every question referenced by the assessment is also publicly shared.
+You may also publicly share whole assessments, provided that every question referenced by the assessment is also publicly shared. When an assessment is shared, the public URL for it can be found on the assessment settings page.
```

Comment:
Aside, no action needed in this PR: this reminds me that we should really add public links to the settings pages for shared questions.